tests/test_app.py

```diff
@@ -50,7 +50,7 @@ def test_invalid_mime_type(self):
         An invalid MIME type is correctly recognized.
         """
         mime_types = [
-            'application/vnd.redhat.insights.advisor+tbz2',
+            # 'application/vnd.redhat.insights.advisor+tbz2',
```

Comment:
I'd just delete this. No sense keeping it if it'll never do anything

Reply:
Yep, agree.